lib/pure/uri.nim

```diff
@@ -311,6 +312,36 @@ proc `$`*(u: Uri): string =
     result.add("#")
     result.add(u.anchor)
 
+proc parseQuery*(query: string): TableRef[string, seq[string]] = 
```

Comment:
Nim's tables support multiple values per key out of the box, no need for the ``seq[string]``. Also it should return an OrderedTableRef since order should be kept. (I now think in general OrderedTable should be the default for everything, see Python's recent dict changes.)